stable/artifactory/templates/artifactory-fluent-config-map.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: v1
```

Comment:
Wrap this file with a big `{{- if .Values.fluentd.enabled }}` too.

Reply:
done